coderadar-graph/src/main/java/io/reflectoring/coderadar/graph/contributor/adapter/ListContributorsAdapter.java

```diff
@@ -32,10 +36,16 @@ public ListContributorsAdapter(
   }
 
   @Override
-  public List<Contributor> listAllByProjectIdAndFilepathInCommit(
-      long projectId, String commitHash, String filename) {
-    return mapper.mapNodeEntities(
-        contributorQueryRepository.findAllByProjectIdAndFilepathInCommit(
-            projectId, commitHash, filename));
+  public List<Contributor> listAllByProjectIdAndPathInCommit(
+      long projectId, String commitHash, String path) {
+    if (fileRepository.fileExistsByPath(projectId, path)) {
+      return mapper.mapNodeEntities(
+          contributorQueryRepository.findAllByProjectIdAndFilepathInCommit(
+              projectId, commitHash, path));
+    } else {
+      return mapper.mapNodeEntities(
```

Comment:
Do we want to throw some kind of error if the directory does not exists/is invalid or just return nothing?